extras/cloudsql/kubernetes-manifests/config.yaml

```diff
@@ -43,8 +43,8 @@ metadata:
 data:
   USE_DEMO_DATA: "True"
   DEMO_LOGIN_USERNAME: "testuser"
-  # All demo user accounts are hardcoded to use the login password 'password'
-  DEMO_LOGIN_PASSWORD: "password"
+  # All demo user accounts are hardcoded to use the login password 'bankofanthos'
+  DEMO_LOGIN_PASSWORD: "bankofanthos"
```

Comment:
Any particular reason why we change this default password? Just curious.

Reply:
Because Chrome (and potentially other browsers) flags `password`, and it does not look good on our prod demo of Bank of Anthos:
![image](https://user-images.githubusercontent.com/3271352/172705061-07c88a05-ac64-45e7-a4b8-ad168d8346ec.png)

`bankofanthos` does not get flagged like this, so it's a much smoother demo experience 😄 

More details here: https://github.com/GoogleCloudPlatform/bank-of-anthos/issues/742